kie-wb/kie-wb-webapp/pom.xml

```diff
@@ -1504,7 +1502,7 @@
             <server>org.jboss.errai.cdi.server.gwt.EmbeddedWildFlyLauncher</server>
             <disableCastChecking>true</disableCastChecking>
             <runTarget>kie-wb.html</runTarget>
-            <hostedWebapp>src/main/webapp</hostedWebapp>
+            <hostedWebapp>${project.build.directory}/kie-wb.war</hostedWebapp>
```

Comment:
I believe that would also be better practice as generated stuff should not go into src/main, but rather target/something.